modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,538 @@
+  --member="<redpanda-console-service-account-email>" \
+  --role='projects/<project-id>/roles/redpanda_console_secret_manager_role'
+```
+
+.. Redpanda Connectors SA:
++
+```bash
+gcloud iam service-accounts create redpanda-connectors \
+  --display-name="Redpanda Connectors Service Account"
+```
+
+* Connectors custom role permissions:
++
+```bash
+gcloud iam roles create redpanda_connectors_role \
+  --project="<project-id>" \
+  --title="Redpanda Connectors Custom Role" \
+  --permissions=resourcemanager.projects.get,secretmanager.versions.access
+```
+
+* Project bindings:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <redpanda-connectors-service-account-email> \
+  --member="<redpanda-connectors-service-account-email>" \
+  --role='projects/<project-id>/roles/redpanda_connectors_role'
+```
+
+* GKE SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+```
+
+. Because some resources can only be created after the Redpanda ID is known, xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] in the https://cloudv2.redpanda.com[Redpanda Cloud UI^] to get the Redpanda ID.
++
+IMPORTANT: On the Network page, select the *Customer-managed* network connection type, and enter the network, service account, and storage bucket information you created. Click *Next*, but before running the `rpk` command on the Deploy page, note the redpanda-id (for example, `cisld88gfi809ee1qjcg`). You finish the cluster deployment in a later step.  
++
+. Use the `redpanda-id` to bind the service accounts with the following roles:
+
+* Redpanda cluster SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]"
+```
+
+* Redpanda Console SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]"
+```
+
+* Connectors SA:
++
+```bash
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<redpanda-id>]"
+```
```

Comment:
suggestion: we are working on [removing the dependency on redpanda-id](https://github.com/redpanda-data/cloudv2/issues/8503). Once it is completed, need to change the section accordingly.